test/e2e/resources/components/ComponentMockFunctions.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<component name="ComponentMockFunctions">
+    <script type="text/brightscript" uri="pkg:/components/scripts/ComponentMockFunctions.brs" />
+    <script type="text/brightscript" uri="pkg:/components/scripts/MockFunctionsUtils-A.brs" />
+    <script type="text/brightscript" uri="pkg:/components/scripts/MockFunctionsUtils-B.brs" />
+    <interface>
+        <field id="json" type="string" />
```

Comment:
It doesn't look like we're using these fields, unless i'm mistaken

Reply:
Whoops, you're right. I initially included them for another purpose but forgot to remove those as the use-case was no longer needed. :)